### PR TITLE
Updatenotifications: Include the scss in the vue component

### PR DESCRIPTION
**Pull request by rullzer**
Quick tests show me that this just works as advertised :)
Also a quick test on IE shows it it still working

### CODE REVIEW

**Comment by nickvergessen** [comment]
But it wont work anymore as soon as one uses variables, right?

**Comment by rullzer** [comment]
We already use variables here. For the text color for example. We expose the css variables already

**Comment by rullzer** [comment]
https://github.com/nextcloud/server/blob/master/apps/updatenotification/css/admin.scss#L27

File: apps/updatenotification/src/components/root.vue [hunk 1]
```diff
@@ -346,3 +346,7 @@
 		}
 	}
 </script>
+
+<style lang="sass" scoped>
+	@import "../../css/admin.scss"
```
Comment (ChristophWurst) [inline apps/updatenotification/src/components/root.vue]: If it's really just used here I wonder if we shouldn't just inline the styles and have this component self-contained in a single file. What do you think?

**Review by ChristophWurst — COMMENTED** [review]
I like it :+1:

File: apps/updatenotification/src/components/root.vue (hunk 1 above)
Comment (rullzer) [reply apps/updatenotification/src/components/root.vue]: Ah yes. Well I was lazy and just did this (in case it did :boom:). So yes then we can inline.

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
Tested and works :+1:

**Pull request merged**